### ISSUE TITLE
Request to increase package size limit for {{name}}

### ISSUE THREAD

**Issue by ibdafna** [jsdelivr/jsdelivr#18322]
**Link to the npm package or GitHub repository**
https://github.com/ibdafna/webdash_dist - pyodide build (that's what I'm request the limit increase for)
https://github.com/ibdafna/webdash - package making use of the files above

**Project license**
 BSD-3-Clause

**Your relation to the project**
Author

**Short description of the project**
WebDash is a Plotly Dash distribution which runs entire in the browser using a WebAssembly Python kernel (Pyodide) and a Flask mock. More details on the repo page linked above.

**Please confirm the following:**
 - [X ] There are no large files that are not necessary to use the package and could be removed (e.g., documentation, examples, tests), or the author is not willing to remove them (please link the related issue in their repo).
 - [X ] The package complies with jsDelivr's [Acceptable Use Policy](https://www.jsdelivr.com/terms/acceptable-use-policy-jsdelivr-net).

**Additional context**
Some of the WASmified Python packages (Scipy, for example), exceed the 20mb threshold and the repo itself probably exceeds the 50mb threshold. However, it is worth noting that not all packages are consumed when an app using WebDash boots - packages are lazy loaded based on whether they're used in the project or not. I did compile the packages with the highest optimization level possible, as well as using dynamically linked libraries where possible - and I was able to get close to 20mb, but not below it. I should also add that I don't think we'll see a lot of traffic on this repo as it's still highly experimental and not used in production.

Thank you!

**Comment by MartinKolarik** [comment]
We already provide a custom endpoint for [Pyodide](https://pyodide.org/en/stable/usage/quickstart.html#try-it-online) with [these packages](https://github.com/pyodide/pyodide/tree/main/packages) - is there anything in your repo that is not available there? If so, would it make sense to add it to the official package store, instead of creating a duplicated repo for it?

**Comment by ibdafna** [comment]
@MartinKolarik Yes, there are a few packages (~6) which are not available in the default Pyodide distribution, but it would not make sense to add them there because they are quite specific to WebDash and would not be useful outside of it. For example, Flask is a web server package which we're patching so that it can "serve" (this is happening inside the browser) a patched version of Plotly Dash. Both of them would not be usable to anyone but WebDash users.

**Comment by MartinKolarik** [comment]
I see. Would it be possible to load only those 6 from your repo (if those are small enough) and the rest from the official distribution? If not, I think we can give you a higher per-file limit, but we don't have that option implemented right now as all the increases before were for npm packages - it would probably take a few days till the limit is increased.

**Comment by ibdafna** [comment]
@MartinKolarik Unfortunately some of the custom files I'm bundling are also hovering above the 20mb mark. Other than that, because of the existing architecture of the Pyodide distribution system, all packages will need to come from a single `indexURL`, making pulling from two separate CDNs non-trivial to implement - hopefully in the future we'll have a more extensible distribution mechanism!

My sincerest apologies for the trouble here :pray: I tried every possible solution before opening this PR. I understand it's a non-trivial request and if it's problematic then I'm happy to start exploring paid options - I know you do your best! If you are going through with the increase then waiting a few days is not an issue!

Many thanks

**Comment by MartinKolarik** [comment]
All right then - just to verify: will 50 MB per file be enough for everything? We probably shouldn't go above that for the GitHub links.

**Comment by ibdafna** [comment]
@MartinKolarik that should be enough. Thank you!
